CharacterCreation/03_Feats.txt

```diff
@@ -921,16 +920,16 @@ Prerequisites:
 *  Hacker or
 *  Knowledge - Nanites at 40
 
-	Add +3 miss chance to a piece of cover for 4 turns.
+	Add -3 accuracy to a piece of cover for 4 turns.

```

Comment:
Scope creep: this feat makes absolutely zero sense and has no flavor text. Who wrote this?